tests/Benchmark_BatchGuesser.cpp

```diff
@@ -0,0 +1,269 @@
+#include <Grid/Grid.h>
+
+using namespace Grid;
+
+template<typename Field> 
+void projAccumulate(const std::vector<Field> &in, std::vector<Field> &out,
```

Comment:
This should be replaced by the guesser function

Reply:
now uses a static class member from the batch guesser module